packages/sdk/utils/src/sign-typed-data-utils.test.ts

```diff
@@ -268,6 +268,169 @@ const TEST_TYPES: EIP712TestCase[] = [
       },
     ],
   },
+  {
+    primaryType: 'AllAtomicTypes',
+    types: {
+      AllAtomicTypes: [
```

Comment:
I know I could have used a loop to generate those, but figured being more explicit was better in this testing context.